README.md

```diff
@@ -512,6 +516,196 @@ get '/', http_codes: [
+property's key is then `entities.<class_key>.<property_name>`. When not found, grape-swagger will
+try to check the its ancestors, up to a class whose name is `Entity`.
+
+Say if there is model class `User` inherits `Grape::Entity`, and another model `AdminUser` inherits
+`User`, to translate a property named `:email` of `AdminUser`, the lookup keys are:
+
+1. `entities.admin_user.email`
+1. `entities.user.email`
+1. `entities.default.email`
+
+#### Example Locale File
+
+```yaml
+en:
+  api:
+    info:
+      title: My Awesome API
+      desc: Some detail information about this API.
+    entities:
+      default:
+        id: Resource identifier
+      user:
+        name: User's real name
+        email: User's login email address
+        sign_up_at: When the user signed up
+      admin_user:
+        level: Which level the admin is
+      password_strength:
+        level: A 0~4 integer indicates `very_weak` to `strong`
+        crack_time: An estimated time for force cracking the password, in seconds
+    params:
+      locale: Used to change locale of endpoint's responding message
+      sort: To specify the order of result list, default is %{default_sort}
+    users:
+      desc: Operations about not-disabled users
+      get:
+        desc: Gets a list of users
+        detail: You can control how to sort the results.
+      ':id':
+        params:
+          id: User id
+        get:
+          desc: Finds user by id
+        email:
+          put:
+            desc: Changes a user's email
+            params:
+              email: A new email
+        password:
+          strength:
+            get:
+              desc: Gets the strength estimation of a user's password
+              detail: The estimation is done by a well-known algorithm when he changed his password
+    swagger_doc:
+      desc: Endpoints for API documents
+      get:
+        desc: Gets root API document
+      ':name':
+        get:
+          desc: Gets specific resource API document
+          params:
+            name: Resource name
+```
+
+### Customization
+
+The translation can be customized by using different kind/format of message in source code.
+
+* A string or `nil`: It will become the default message when a translation is not defined.
+* A symbol: Looks up translation using the symbol as a key, but will fallback to default key(s).
+* A hash may contain these information:
+  * key: A symbol, defines custom lookup key.
+  * default: A string, defines the default message when translation can not be found.
+  * bypass: A boolean (default is `false`), set to `true` to bypass translation looking up for this
+    message, use the `:default` value or blank string directly.
+  * scope: A symbol, or a string, or an array of them. Used to replace the `i18n_scope` config
+    value for a custom lookup scope.
+  * Any others: Custom variables which will be sent to the translation for interpolation.
```

Comment:
Say "Any other key/value will be sent to the translation function for interpolation."